src/Message/AuthorizeRequest.php

```diff
@@ -7,6 +7,21 @@
  */
 class AuthorizeRequest extends AbstractRequest
 {
+    public function getApplicationFee()
+    {
+        return $this->getParameter('applicationFee');
+    }
+
+    public function getApplicationFeeInteger()
+    {
+        return (int) round($this->getApplicationFee() * pow(10, $this->getCurrencyDecimalPlaces()));
```

Comment:
I see what you mean now. I forgot we were doing that for amount. I will probably refactor some of that out for V3, but for the time being this can be left as is, so it's consistent.